athletics.lic

```diff
@@ -66,6 +66,15 @@ class Athletics
     stow_athletics_items
   end
 
+  def outdoorsmanship_waiting(n)
+    start = Time.now
```

Comment:
Since the purpose of this script is to train athletics, some folks may not understand why the script pauses and begins training outdoorsmanship between loops. I'm suggesting a couple echos to explain that a wait is needed and if the script pauses at the end for any amount of time so they don't think something's wrong.
```suggestion
    echo "Need to wait a bit before resume athletics training"
    start = Time.now
```

Example output:
```
[athletics: Need to wait a bit before resume athletics training]

--- Lich: outdoorsmanship active.

[outdoorsmanship]>collect rock

With skill and nimble fingers you avoid the centipede that runs along in front of your fingers.
You manage to collect a pile of rocks.
Roundtime: 15 sec.
>
You take a step back and run up to the pile of rocks.  Bringing your foot smashing down onto the pile, you send bits and pieces of debris scattering in every direction.  Looking around for the now defunct mountain of glory, you fail to find any signs of its presence.

...
 
--- Lich: outdoorsmanship has exited.

[athletics: Almost ready, need to wait 11 seconds before restart athletics training]
... 
```